providers/aws/fargate/cluster.go

```diff
@@ -289,3 +293,45 @@ func (c *Cluster) RemovePod(tag string) {
 
 	delete(c.pods, tag)
 }
+
+// GetContainerLogs returns the logs of a container from this cluster.
+func (c *Cluster) GetContainerLogs(namespace, podName, containerName string, tail int) (string, error) {
+	prefix := fmt.Sprintf("%s_%s", buildTaskDefinitionTag(c.name, namespace, podName), containerName)
+
+	describeResult, err := client.logs.DescribeLogStreams(&cloudwatchlogs.DescribeLogStreamsInput{
+		LogGroupName:        aws.String(c.cloudWatchLogGroupName),
+		LogStreamNamePrefix: aws.String(prefix),
+	})
+	if err != nil {
+		return "", err
+	}
+
+	// Nothing logged yet
+	if len(describeResult.LogStreams) == 0 {
+		return "", nil
+	}
+
+	logs := ""
+
+	err = client.logs.GetLogEventsPages(&cloudwatchlogs.GetLogEventsInput{
+		Limit:         aws.Int64(int64(tail)),
+		LogGroupName:  aws.String(c.cloudWatchLogGroupName),
+		LogStreamName: describeResult.LogStreams[0].LogStreamName,
```

Comment:
There shouldn't be more then one log stream per container, but maybe we should guard this better?